### PR TITLE
Align breakpoint line numbers

### PR DESCRIPTION
**Pull request by jtpio**
A quick pass on the breakpoints UI to:

- align the line numbers to the right
- handle the overflow for long source path
- add title attribute with the full path

image

### CODE REVIEW

**Comment by afshin** [comment]
This is strictly an improvement, so let's merge it.

I think a further enhancement could be applied to the truncating of the file name to truncate at the beginning instead of at the end, like the JupyterLab file browser:

Screen Shot 2019-12-09 at 12 22 16 PM

File: src/breakpoints/body.tsx [hunk 1]
```diff
@@ -94,10 +94,11 @@ const BreakpointComponent = ({
     <div
       className={`breakpoint`}
       onClick={() => model.clicked.emit(breakpoint)}
+      title={breakpoint.source.path}
     >
-      <span>
-        {breakpoint.source.path} : {breakpoint.line}
-      </span>
+      <span className={'marker'}>●</span>
```
Comment (afshin) [inline src/breakpoints/body.tsx]: Are the class names `marker` and `source` intentionally not prefixed?

Comment (jtpio) [reply src/breakpoints/body.tsx]: For now yes, as it follows the existing style (we should also change the `breakpoint` class from the wrapper `<div>`).

We will have to do a proper pass anyway with #139, but happy to make the here too.

**Comment by jtpio** [comment]
That would probably be better indeed.

We'll have to see if it makes sense to do that now, since there were a couple of ideas and questions for an alternative UI breakpoint (https://github.com/jupyterlab/debugger/issues/171 and others).

**Pull request merged**